tests/application-test.js

```diff
@@ -551,6 +551,35 @@ describe('Box.Application', function() {
 
 			});
 
+
+			it('should not be called on behaviors in when stopImmediatePropagation() is called', function() {
```

Comment:
I'd like to add another test for when behavior1 calls stopImmediatePropagation() and causes behavior2 to not fire